tools/wt_verify/wt_verify.py

```diff
@@ -125,7 +126,8 @@ def parse_node(f, line, output, chkpt_info, root_addr, is_root_node):
     node = {}
     line = line[2:-1] # remove new node symbol
     page_type = line.split()[-1]
-    assert page_type == "internal" or page_type == "leaf"
+    if not page_type in ("internal", "leaf"):
+        raise Exception(f"page_type expected to be 'internal' or 'leaf' but found '{page_type}'")
```

Comment:
This suggests you create a variable that contains the two possible values.
```suggestion
        raise Exception(f"page_type expected to be one of the values from {<variable_here>} or 'leaf' but found '{page_type}'")
```